src/reporters/json.ts

```diff
@@ -252,26 +364,7 @@ export default class JSONReporter extends BaseReporter {
 
     this.runner.on(
       'step:end',
-      ({
-        journey,
-        step,
-        start,
-        end,
-        error,
-        screenshot,
-        url,
-        status,
-        metrics,
-      }) => {
-        if (screenshot) {
-          this.writeJSON({
-            type: 'step/screenshot',
-            journey,
-            step,
-            blob: screenshot,
-            blob_mime: 'image/jpeg',
-          });
-        }
+      ({ journey, step, start, end, error, url, status, metrics }) => {
         this.writeJSON({
```

Comment:
I'm wondering if we should add a new field to the payload `screenshot_type: "ref"` to make it easier on the client to know how to handle the screenshot. WDYT @justinkambic ?